src/core/qgslabelfeature.cpp

```diff
@@ -78,6 +78,15 @@ QgsFeature QgsLabelFeature::feature() const
   return mFeature;
 }
 
+QSizeF QgsLabelFeature::size( double angle ) const
+{
+  if ( mRotatedSize.isEmpty() )
+    return mSize;
+
+  // Between 45 to 135 and 235 to 313 degrees, return the rotated size
```

Comment:
Forgive my ignorance: why 313 and not 315?

Reply:
Good catch, it's a typo, should be 315.

Reply:
indeed I see the right number in code later.